resources/check-ts.js

```diff
@@ -0,0 +1,19 @@
+// @noflow
```

Comment:
@IvanGoncharov I couldn't figure out a way to point `dtslint` at `src` and have it look for the `tsconfig.json` in the root. Pointing it at the root also doesn't work because it explicitly looks for an `index.d.ts` file in whatever dir you set. TL;DR This is a bandaid solution to keep `tsconfig.json` in the root without duplicating it inside `src`.

Once we've converted everything to TS, we'll be able to drop `dtslint` altogether and we can remove this script.

Reply:
@danielrearden We need `dtslint` in the long run to ensure that we ship correct typings that work with older versions of typescript since https://github.com/sandersn/downlevel-dts is not really mature yet.
Better solution would be to build npm package into some tmp directory and run dtslint against this temp dir.

Reply:
@IvanGoncharov Boo 😭  Welp, there's always monkey-patching.

On a related note, maybe you were already planning to do this, but we could [publish separate files targeting different TS versions](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-1.html#version-selection-with-typesversions) for the type declarations.

Reply:
> On a related note, maybe you were already planning to do this, but we could publish separate files targeting different TS versions for the type declarations.

yes, it's what the TS team wants library authors to do with `downlevel-dts`.